core/src/main/scala/chisel3/Bits.scala

```diff
@@ -121,12 +127,12 @@ sealed abstract class Bits(private[chisel3] val width: Width) extends Element wi
     *
     * @param x an index
     * @return the specified bit
-    * @note convenience method allowing direct use of [[scala.Int]] without implicits
     */
-  final def apply(x: Int): Bool = macro SourceInfoTransform.xArg
+  final def apply(x: Int): Bool = macro IntLiteralApplyTransform.safeApply
 
-  /** @group SourceInfoTransformMacro */
-  final def do_apply(x: Int)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): Bool = apply(BigInt(x))
+  /** @group IntLiteralApplyTransformMacro */
```

Comment:
```suggestion
  /** @group SourceInfoTransformMacro */
```
This group is not related to the actual `macro` class, it's purely used for ScalaDoc grouping, leave that bit alone.